onnx/defs/schema.h

```diff
@@ -1416,21 +1475,28 @@ class OpSchemaRegistry final : public ISchemaRegistry {
     for (auto& x : map()) {
       for (auto& y : x.second) {
         auto& version2schema = y.second;
-        r.emplace_back(version2schema.rbegin()->second);
+        if (!version2schema.empty()) {
+          r.emplace_back(version2schema.rbegin()->second);
+        }
       }
     }
     return r;
   }
 };
 
-void RegisterSchema(OpSchema schema, int opset_version_to_load = 0, bool fail_duplicate_schema = true);
+void RegisterSchema(
```

Comment:
This is a public interface we need to maintain. I think we need an `void RegisterSchema(OpSchema schema, ...` overload?

Reply:
`OpSchema &&` and `OpSchema` are ambiguous during compilation. I've added `const OpSchema&` to handle lvalues.